.github/workflows/build_worker_images.yml

```diff
@@ -0,0 +1,38 @@
+name: Build default worker images
+on:
```

Comment:
Can you add a comment here explaining when the default worker images are built, and why this decision was made?